### ISSUE TITLE
How can I upload a pandas dataframe to redshift?

### ISSUE THREAD

**Issue by mandywoo** [aws/amazon-redshift-python-driver#32]

**Comment by Brooke-white** [comment]
Hi @mandywoo ,

Thank you for reaching out. Pandas dataframes can be written to an Amazon Redshift table using `cursor.write_dataframe()`. 

A usage example can be found in our unit test suite, [here](https://github.com/aws/amazon-redshift-python-driver/blob/31ffbb5d0e6d3cd8628d1d8139e7a9aab96bfb0e/test/integration/test_pandas.py#L60).

The doc-string for `write_dataframe()` can be found [here](https://github.com/aws/amazon-redshift-python-driver/blob/fbc9b5a0ac8332e219d09885b035a81736bde786/redshift_connector/cursor.py#L379).

In short, the first parameter to this method is a `Pandas.DataFrame` and the second is a `str`, the name of the table. The `Pandas.DataFrame` column names should be inline with the column names of the Amazon Redshift table for insertion, as should the dimensions of the data.

I have created a note in our internal tracking system to include details of this feature in the project's README. Please reach out if any clarification is needed.
